servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -2166,9 +2169,7 @@ void RenderForwardMobile::_render_list_template(RenderingDevice::DrawListID p_dr
 			} break;
 		}
 
-		PipelineCacheRD *pipeline = nullptr;
-
-		pipeline = &shader->pipelines[cull_variant][primitive][shader_version];
+		PipelineCacheRD *pipeline = &shader->pipelines[cull_variant][primitive][shader_version];
```

Comment:
Just as a heads up, we will have a merge conflict on this case as this code will be replaced in #90400. Seeing as it's for aesthetic reasons I'd forego this change and the removal of the comment below.

Reply:
Ugh. This is exactly why I included it (so someone could notice it, it wasn't an aesthetic change but more like bait).

There are more upcoming changes that will affect `RenderForwardMobile::_render_list_template`, as there is an optimization that uses Spec Constants for key in sorting the draw calls for better draw call batching.

Reply:
I will probably have to ask for your help when the time comes.